nsq.h

```diff
@@ -15,15 +15,11 @@
 struct Buffer; // from evbuffsock.h
 
 typedef enum {NSQ_FRAME_TYPE_RESPONSE, NSQ_FRAME_TYPE_ERROR, NSQ_FRAME_TYPE_MESSAGE} frame_type;
-typedef enum {NSQ_PARAM_TYPE_INT, NSQ_PARAM_TYPE_CHAR} nsq_cmd_param_type;
+typedef enum {NSQ_PARAM_TYPE_INT, NSQ_PARAM_TYPE_CHAR} nsqCmdParamType;
+typedef enum {NSQ_LOOKUPD_MODE_READ, NSQ_LOOKUPD_MODE_WRITE} nsqLookupdMode;
```

Comment:
this is weird, because you should not use lookupd for a nsq writer (publisher), only for a nsq reader (consumer)

Reply:
I mean to use lookupd to select one nsqd_connection for producer (it seems no need to do this job when the topic_name does not exists at the first time), the consumer also does the same thing. OK, I remove this code firstly.